demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -1968,12 +1969,60 @@ def validate_job(self, structure_validator, pack_error_ignore_list):
 
         return is_valid
 
+    def get_all_files_edited_in_pack_ignore(self, modified_files):
+        all_files = self.git_util.get_all_files()
+        all_files_edited_in_pack_ignore = set()
+        for file_path in modified_files:
+            # handle renamed files
+            if isinstance(file_path, tuple):
+                old_file_path = file_path[0]
+                file_path = file_path[1]
+
+            else:
+                old_file_path = file_path
+            if not file_path.endswith(".pack-ignore"):
+                continue
+            old_file_content = get_remote_file(old_file_path, tag="master")
+            config = ConfigParser(allow_no_value=True)
+            config.read_string(old_file_content)
+            old_file_content = self.get_error_ignore_list(config=config)
+            pack_name = get_pack_name(str(file_path))
+            file_content = self.get_error_ignore_list(pack_name)
+            files_to_test = set()
+            for key, value in old_file_content.items():
+                if not (section_values := file_content.get(key, [])) or not set(
+                    section_values
+                ) == set(value):
+                    files_to_test.add(key)
+            for key, value in file_content.items():
+                if not (section_values := old_file_content.get(key, [])) or not set(
+                    section_values
+                ) == set(value):
+                    files_to_test.add(key)
+
+            def is_relevant_file(file, pack_name):
+                return len(file.parts) > 2 and file.parts[1] == pack_name
```

Comment:
what are 2 and 1? Best not to use magic numbers

Reply:
Would this do?
```suggestion
                return len(file.parts) > 2 and file.parts[0] == "Packs" and file.parts[1] == pack_name
```

Reply:
im saying that what does it mean the file.parts > 2 ? and what is the first/second part?

Reply:
The given file argument is some file_path from content repo.
We want to ensure that this file is related to the relevant pack so we're checking that it is under Packs/pack_name

Reply:
I'll add the other connection and some doc strings.